src/status_im/react_native/resources.cljs

```diff
@@ -52,7 +52,21 @@
    :community-cover    (js/require "../resources/images/ui/community-cover.png")
    :lifestyle          (js/require "../resources/images/ui/lifestyle.png")
    :music              (js/require "../resources/images/ui/music.png")
-   :podcasts           (js/require "../resources/images/ui/podcasts.png")})
+   :podcasts           (js/require "../resources/images/ui/podcasts.png")
+   :placeholder-image1 (js/require "../resources/images/ui/placeholder-image1.png")
```

Comment:
could you elaborate why do we need these images ?

Reply:
@flexsurfer  These images are simply serving as a placeholder  at the moment and are used on the discover-card component but as I found out from design this component is not yet done with design but for the time being untill they are done with illustrations which is like in 2-3 months we just have to do with the placeholder images on this component.

Reply:
cool will just do that. thanks